http/src/main/scala/Accepts.scala

```diff
@@ -16,13 +16,14 @@
 //: ----------------------------------------------------------------------------
 package nelson
 
+import cats.effect.IO
 import org.http4s._
 
 /** This should be added to http4s */
 abstract class Accepts(mediaType: MediaType) {
-  def unapply(req: Request): Boolean =
+  def unapply(req: Request[IO]): Boolean =
     req.headers.get(headers.Accept).fold(true)(
-      _.values.list.exists(_.mediaRange.satisfiedBy(mediaType)))
+      _.values.toList.exists(_.mediaRange.satisfiedBy(mediaType)))
```

Comment:
Doesn't `Foldable` have `exists` now?  Not that it's a showstopper.

Reply:
`scalaz.NonEmptyList` so its either here or define the instance in `CatsHelpers`. Since eventually it's going to become `cats.data.NonEmptyList` I figure I'd touch the minimal amount of stuff